scripts/wsrep_sst_xtrabackup-v2.sh

```diff
@@ -1476,11 +1471,24 @@ send_data_from_donor_to_joiner()
 
     pushd ${dir} 1>/dev/null
     set +e
-    timeit "$msg" "$strmcmd | $tcmd; RC=( "\${PIPESTATUS[@]}" )"
+
+
+    # Implement retry logic ourselves, as nc needs it, and socat has a bug
```

Comment:
Here we've got hardcoded 30. Previously it was configurable via `retry` config parameter. Maybe it would be better to detect the value of `retry` in `read_cnf()` and use it here in the loop?

Reply:
It was configurable, but not documented, and I didn't see any practical use in our uses, so I'm not sure if that's really needed.

Also this is how the 8.0 code works for nc, with a hardcoded loop up to 30, ignoring the retry value.